### ISSUE TITLE
If insert a Row after add an ConditionalFormat, the ConditionalFormat disappears

### ISSUE THREAD

**Issue by InaBowBow** [ClosedXML/ClosedXML#389]
**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**What is the current behavior?**
First, set ConditionalFormat for the cell. After that, when you insert some rows, ConditionalFormat disappears.
If Row is not inserted, ConditionalFormat will remain.

**What is the expected behavior or new feature?**
ConditionalFormat will remain even if Row is inserted

**Code to reproduce problem:**
```c#
public void Main()
{
  using (var wb = new XLWorkbook(@"C:\test.xlsx")) {
    var ws = wb.Worksheet("sheet1");

    // Add ConditionalFormat
    ws.Cell("A2").AddConditionalFormat()
      .WhenNotEquals("test")
      .Font.SetFontColor(XLColor.Red);

    // Insert Row
    ws.Row(1).InsertRowsBelow(1);

    // Save Excel
    wb.Save();
  }
}
```

**Comment by igitur** [comment]
Try apply the changes from #343 to see if it solves your issue.

**Comment by Pankraty** [comment]
On current `develop` branch and a new file (as no `test.xlsx` was attached) I get this result:
![image](https://user-images.githubusercontent.com/19576939/40273598-025c556e-5bd4-11e8-8de5-8cbf98d63fa1.png)
[389_result.xlsx](https://github.com/ClosedXML/ClosedXML/files/2019835/389_result.xlsx)

I've no idea when this was fixed, but let's assign a 0.93 milestone.
